pkg/ucp/queue/inmemory/client.go

```diff
@@ -44,12 +44,17 @@ func New(queue *InmemQueue) *Client {
 
 // New creates the named in-memory queue Client instance.
 func NewNamedQueue(name string) *Client {
-	inmemq, _ := namedQueue.LoadOrStore(name, NewInMemQueue(messageLockDuration))
+	inmemq, _ := namedQueue.LoadOrStore(name, NewInMemQueue(name, messageLockDuration))
 	return &Client{
 		queue: inmemq.(*InmemQueue),
 	}
 }
 
+// String returns the name of the queue for diagnostics purposes.
+func (c *Client) String() string {
```

Comment:
Should this function be called **Name**?

Reply:
This has been removed